docs/dotnet/dotnet-dns-metrics.md

```diff
@@ -0,0 +1,51 @@
+# Semantic Conventions for DNS metrics emitted by .NET
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for DNS metrics emitted by .NET.
+
+**Disclaimer:** These are initial .NET metric instruments available in .NET 8 but more may be added in the future.
+
+<!-- toc -->
+
+- [DNS metrics](#dns-metrics)
+  * [Metric: `dns.lookups.duration`](#metric-dnslookupsduration)
+
+<!-- tocstop -->
+
+## DNS metrics
+
+### Metric: `dns.lookups.duration`
+
+**Status**: [Experimental][DocumentStatus]
+
+This metric is required.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/metrics/api.md#instrument-advice)
+of **`[TODO]`**.
+
+<!-- semconv metric.dotnet.dns.lookup.duration(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `dns.lookup.duration` | Histogram | `s` | Measures the time take to perform a DNS lookup. [1] |
+
+**[1]:** Meter name is `System.Net.NameResolution`.
+<!-- endsemconv -->
+
+<!-- semconv metric.dotnet.dns.lookup.duration(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `question.name` | string | The name being queried. [1] | `www.example.com`; `dot.net` | Required |
```

Comment:
I believe it was suggested to change this to dns.question.name, which sounds good to me.

Reply:
fixed